index.html

```diff
@@ -5666,6 +5788,13 @@ <h3>Storage Providers and Data Mining</h3>
 expectations.
           </li>
         </ul>
+
+        <p>
+In addition to the mitigations above, the participation of civil society and
+regulators in the analysis and auditing of vendors can also ensure that there
+are legal protections in place, and enforced, for individuals affected by
+practices that are not aligned with their best interests.
+        </p>
```

Comment:
This is far from actionable specification text for implementers.  It's closer to sociology.

Please delete this unnecessary addition.

Reply:
This text was explicitly requested by @jyasskin in his review. It's in a privacy considerations section, and it's important to point out that people like @npdoty need to have a say in oversight wrt. digital credentials ecosystems.